src/main/java/com/voicepin/example/client/messages/VerifyResponse.java

```diff
@@ -4,10 +4,10 @@
 
 public class VerifyResponse extends Response {
 
-    @JsonProperty("Score")
+    @JsonProperty("score")
```

Comment:
Chyba nie ma sensu trzymać stringa w adnotacji skoro pole nazywa się tak samo. Niżej to samo.